Lib/test/test_glob.py

```diff
@@ -521,7 +521,6 @@ class SymlinkLoopGlobTests(unittest.TestCase):
     # gh-109959: On Linux, glob._isdir() and glob._lexists() can return False
     # randomly when checking the "link/" symbolic link.
     # https://github.com/python/cpython/issues/109959#issuecomment-2577550700
-    @unittest.skip("flaky test")
```

Comment:
No, this one should remains flaky. What I meant is that you can use the `requires_resource()` decorator *or* leave this file alone.